source/opt/libspirv.hpp

```diff
@@ -24,31 +24,35 @@
 // TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 // MATERIALS OR THE USE OR OTHER DEALINGS IN THE MATERIALS.
 
-#ifndef LIBSPIRV_TEST_OPT_OPT_TEST_COMMON_H_
-#define LIBSPIRV_TEST_OPT_OPT_TEST_COMMON_H_
+#ifndef SPIRV_TOOLS_LIBSPIRV_HPP_
+#define SPIRV_TOOLS_LIBSPIRV_HPP_
 
 #include <memory>
 #include <string>
 #include <vector>
 
-#include "source/opt/module.h"
+#include "module.h"
+#include "spirv-tools/libspirv.h"
 
 namespace spvtools {
-namespace opt {
 
-// TODO(antiagainst): expand and export these functions as the C++ interface in
-// libspirv.hpp.
+// TODO(antiagainst): add parameters for target environment and error message.
```

Comment:
Let's add the context object before making this API public.
I think it should be spvtools::ir::Context  and then the Assemble, Disassemble, and BuildModule functions are methods on the context object.